app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    def inner(*args) -> tuple:
```

Comment:
```suggestion
    def inner(*args, **kwargs) -> tuple:
```
You should consider that the function can accept keyword arguments as well.